src/components/EntityInfo.vue

```diff
@@ -39,66 +39,73 @@
               />
             </span>
 
-            <!-- View and Change Company Information -->
-            <span v-if="showViewChangeInfoBtn">
-              <v-btn
-                small text color="primary"
-                id="company-information-button"
-                :disabled="hasBlocker || !isInGoodStanding"
-                @click="viewChangeCompanyInfo()"
-              >
-                <v-icon medium>mdi-file-document-edit-outline</v-icon>
-                <span class="header-action-text">View and Change Company Information</span>
-              </v-btn>
-
-              <v-tooltip top content-class="pending-tooltip" v-if="isPendingDissolution || isNotInCompliance">
-                <template v-slot:activator="{ on }">
-                  <span class="pending-alert pr-2" v-on="on">
-                    <v-icon color="orange darken-2">mdi-alert</v-icon>
-                  </span>
-                </template>
-                <template v-if="isPendingDissolution">
-                  You cannot view or change company information while the company is pending dissolution.
-                </template>
-                <template v-if="isNotInCompliance">
-                  You cannot view or change company information while the company is not in compliance.
-                </template>
-              </v-tooltip>
+            <span v-if="isHistorical">
+              <v-chip class="primary mt-n1 ml-4" small label text-color="white">HISTORICAL</v-chip>
+              <span class="font-14 ml-3">{{dissolutionText}}</span>
             </span>
 
-            <!-- Dissolve Company -->
-            <span v-if="isDissolutionEnabled" class="pl-1">
-              <v-btn
-                small text color="primary"
-                id="dissolution-button"
-                :disabled="hasBlocker"
-                @click="prompDissolve()"
-              >
-                <img src="@/assets/images/Dissolution_Header_Icon.svg" alt="" class="pr-2">
-                <span class="header-action-text">Dissolve this {{ entityDescription }}</span>
-                <v-tooltip top content-class="top-tooltip" transition="fade-transition" nudge-right="7">
+            <template v-else>
+              <!-- View and Change Company Information -->
+              <span v-if="showViewChangeInfoBtn">
+                <v-btn
+                  small text color="primary"
+                  id="company-information-button"
+                  :disabled="hasBlocker || !isInGoodStanding"
+                  @click="viewChangeCompanyInfo()"
+                >
+                  <v-icon medium>mdi-file-document-edit-outline</v-icon>
+                  <span class="font-13 ml-1">View and Change Company Information</span>
```

Comment:
I tweaked how the different buttons get spaced out. This is now more consistent / no special cases.

Reply:
Sounds good! Just to double check the icons still still align with the headers above? *see pic
Tracey was particular about his in a recent review

![Screen Shot 2021-11-05 at 8 05 44 AM](https://user-images.githubusercontent.com/53542131/140532717-cf148027-0db3-499b-993a-c8977043002b.png)
.

Reply:
LGTM:

![image](https://user-images.githubusercontent.com/10002889/140542052-b9f44581-0c5c-409b-ba98-1862a46582f9.png)